README.md

```diff
@@ -3,8 +3,43 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. And after a year and a half, we're making sure that the direction we're heading in is clear to everyone who wants to know.
 
-We _very much_ want/need help on this so if you have _time_
+### Our Goal/Mission
+
+> Empower people to sustainably unlock their potential and pursue their passion through technology.
```

Comment:
Entrepreneurs often put emphasis on passion because passion helps fuel motivation to push through difficult situations, etc. However successful entrepreneurs are also often good at what they are passionate about (so there is a selection bias here). There are many many failed entrepreneurs who might have been passionate about something, but weren't good at it and therefore failed. The fundamental nature of `do what you love` doesn't make sense and it should be based around what skills you have and can apply (purpose?) and utilising them to do great things and therefore more of a `love what you do` thing